aibolit/patterns/string_concat/string_concat.py

```diff
@@ -28,7 +28,7 @@ def value(self, filename: str) -> List[LineNumber]:
                 is_l_this = isinstance(node.operandl, javalang.tree.This)
                 is_r_this = isinstance(node.operandr, javalang.tree.This)
                 if is_l_literal and (is_r_member or is_r_meth_inv or is_r_this):
-                    is_string_literal = '"' in node.operandl.value
+                    is_string_literal = '"' in node.operandl.value  # type: ignore
```

Comment:
why type ignore?

Reply:
because mypy thinks that it doesn't have such a value